app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -86,4 +90,23 @@ open class FocusApplication : LocaleAwareApplication(), CoroutineScope {
         StrictMode.setThreadPolicy(threadPolicyBuilder.build())
         StrictMode.setVmPolicy(vmPolicyBuilder.build())
     }
+
+    private fun initializeWebExtensionSupport() {
+        try {
```

Comment:
If I remove this all unit tests from BrowserMenuController are failing with java.lang.UnsupportedOperationException: Web extension support is not available in this engine. I will get documented about it

Reply:
We probably want to mock that API out in the unit tests I guess. Since we know in the real engine this is a valid API, but the default implementation is to throw.